pyproject.toml

```diff
@@ -2,7 +2,7 @@
 
 [tool.poetry]
 name = "aerosense-dashboard"
-version = "0.4.4"
+version = "0.5.0"
```

Comment:
I think this should be version `0.4.5` as no new features or breaking changes have been added